operations/phlare/helm/phlare/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "phlare.fullname" . }}-ingress
+  labels:
+    {{- include "phlare.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  rules:
+  {{- range $.Values.ingress.hosts }}
+  - host: {{ . | quote }}
+    http:
+      paths:
+      - backend:
+          service:
+            {{- if gt (len $.Values.phlare.components) 1}}
+            name: {{ include "phlare.fullname" $ }}-query-frontend
+            {{- else }}
+            name: {{ include "phlare.fullname" $ }}
+            {{- end }}
+            port:
+              number: 4100
```

Comment:
Should this be the service port configuration instead of 4100 ? `{{.Values.service.port}}`

Reply:
There's no way though to change the default port, so this is really just a neat but I think we should probably fix that later.